src/utils/use-title.ts

```diff
@@ -0,0 +1,47 @@
+import { useEffect } from 'react'
+import { useParams } from 'react-router-dom'
+
+const setTitle = (title: string) => {
+  document.title = title
+}
+
+export const useTitle = (customString?: string) => {
```

Comment:
```suggestion
export const useTitle = (pagePrefix?: string) => {
```
My thought process here is that anything after the `Lora - ` is the page specific portion of the title, so it's a page prefix, even though it's not the title prefix.